finality-aleph/src/network/manager/discovery.rs

```diff
@@ -498,181 +344,32 @@ mod tests {
             ) if rebroadcast_authentication == &authentication)));
     }
 
-    #[tokio::test]
-    async fn responds_to_correct_request_when_can() {
```

Comment:
Btw why do we remove some tests and adapt others? Test coverage is the same right, we just remove tests that covered functionalities no longer exist.

Reply:
It is exactly what I did. I had to delete every test that was connected to`DiscoveryMessage::Request` and adapt those connected to `DiscoveryMessage::Authentication(s)`